### PR TITLE
feat: Updated color ramps

### PR DESCRIPTION
**Pull request by ShrimpCryptid**
Problem
=======
Changes the default color ramp to match publications (https://github.com/allen-cell-animated/nucmorph-colorizer/issues/67) and reverses color ramps for consistency (https://github.com/allen-cell-animated/nucmorph-colorizer/issues/71).

Solution
========
- Added a new color ramp for Matplotlib cool, and set it as the default ramp.
- Reversed some color ramps so they are perceptually lighter for lower values and darker for higher values.
- Added names for each of the color ramps in anticipation of a future feature (https://github.com/allen-cell-animated/nucmorph-colorizer/issues/82).
  - This was originally an optional part of #71 but I decided to spin it off as a new issue.
  
Note: Sorry the commit history is crazy long! I branched this off of another feature but am a little scared to rebase it back onto main.

## Type of change
* New feature (non-breaking change which adds functionality)

Screenshots (optional):
-----------------------
Before:
image

After:
image

### CODE REVIEW

**Review by toloudis — APPROVED** [review]
I really like having the string names in there.

It's not clear which color ramps got reversed.  How did you choose which ones? The original ticket didn't specify which ones to flip.     Also we should create a ticket for the "flip color ramp" checkbox option! :)

**Comment by ShrimpCryptid** [comment]
> I really like having the string names in there.
> 
> It's not clear which color ramps got reversed. How did you choose which ones? The original ticket didn't specify which ones to flip. Also we should create a ticket for the "flip color ramp" checkbox option! :)

I updated the PR description to include the before and after!

**Review by frasercl — APPROVED** [review]
Good to see more useful color ramps coming in! Again, the ones I added are pretty provisional, so we don't need to be precious with them. It might make sense to sit down with a scientist or few eventually and see if the list can be pared down, if not swapped out entirely for ramps like this one from tools they're used to.

File: src/constants.tsx [hunk 1]
```diff
@@ -2,50 +2,58 @@ import { HexColorString } from "three";
 import { ColorRamp } from "./colorizer";
 
 // https://developers.arcgis.com/javascript/latest/visualization/symbols-color-ramps/esri-color-ramps/
-const colorStops: HexColorString[][] = [
+const colorStops: [string, HexColorString[]][] = [
+  // Matplotlib - cool
+  ["matplotlib-cool", ["#00ffff", "#ff00ff"]],
```
[the remark this answers is not in the document]
Comment (ShrimpCryptid) [reply src/constants.tsx:7]: I originally structured it like this with the intention of building a Map object down the line to map from names to Color Ramps in an ordered way! This is definitely subject to change, but I wanted to at least get something in the codebase while I was working on the file.

**Pull request merged**